revenuecat_examples/MagicWeather/android/proguard-rules.pro

```diff
@@ -0,0 +1,27 @@
+# Add project specific ProGuard rules here.
```

Comment:
This file should go in the `app` folder like in purchase_tester right?

Reply:
You're right!